docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Host and deploy Blazor
+author: guardrex
+description: Discover how to host and deploy Blazor apps using ASP.NET Core, Content Delivery Networks (CDN), file servers, and GitHub Pages.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/host-and-deploy/index
+---
+# Host and deploy Blazor
+
+By [Luke Latham](https://github.com/guardrex) and [Rainer Stropek](https://www.timecockpit.com)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+## Publishing
+
+Blazor apps are published for deployment in Release configuration with the [dotnet publish](https://docs.microsoft.com/dotnet/core/tools/dotnet-publish) command:
+
+```console
+dotnet publish -c Release
+```
+
+The .NET linker strips unused [Intermediate Language (IL)](https://docs.microsoft.com/dotnet/standard/managed-code) when the app is built in Release configuration.
```

Comment:
Instead of saying it strips unused IL, could we say it strips unused *methods*? (which BTW is the actual level of granularity it works on)

Example:

As part of the build process, the .NET linker removes any unused methods and assemblies. This makes the resulting application payload much smaller than it would have been if the entire .NET Base Class Library (BCL) was packaged with it.

Reply:
The link is actually run on every build, not just on publish. I would also prefer not to use the made up name ".NET linker". Instead how about: "When the app is published it will first be built (if it hasn't been already). As part of the build any used methods and assemblies will be removed to reduce app download size and speed up load times."